File: HackerEarth Algo Problem/Basics of InputOutput/6-Palindromic String/TestClass.py
s=input()
l1=list(s)
l=len(l1)
o=l-1
c=0
for i in range(0,len(l1)):
    #print(l1[i])
    #print(l1[o])
    if l1[i]==l1[o]:
        c=c+1
    o=o-1

if c==l:
    print("YES")
else:
    print("NO")

File: HackerEarth Algo Problem/Basics of InputOutput/6-Palindromic String/TestClass.c
#include<stdio.h>
#include<string.h>

int main()
{
char str[100];
gets(str);

int j=0,c=0;
int l=strlen(str);
  for(int i=l-1;i>=0;i--)
  {
	  
	  if(str[j]==str[i])
	  {
		  c++;
	  }
	  j++;
	  
  }
   if(c==l)
  {
	  printf("YES");
  }
  else
  {
	  printf("NO");
  }

return 0;
}